wee_slack.py

```diff
@@ -2587,8 +2602,33 @@ def subprocess_channel_leave(message_json, eventrouter, channel, team):
 
 
 def subprocess_message_replied(message_json, eventrouter, channel, team):
-    pass
-
+    if "message" not in message_json:
```

Comment:
Can this happen?

Reply:
I guess not but I couldn’t find a real specification for it in the slack api documentation

Reply:
[The example in the docs](https://api.slack.com/events/message/message_replied) includes it, and the [docs describing threads](https://api.slack.com/docs/message-threading) describes what the message will contain, so I think it's safe to assume it will always be there.

Reply:
✓